Assets/Scenes/ResourceDisplay.unity~HEAD~18

```diff
@@ -0,0 +1,487 @@
+%YAML 1.1
```

Comment:
You probably want this file instead of ` Assets/Scenes/ResourceDisplay.unity.meta`. Delete that one and rename this one to that